install.sh

```diff
@@ -1067,17 +1097,18 @@ questionaire() {
     echo -e "(Note that all this script does is set a lot of the time consuming parameters in the config"
     echo
     echo -e "${PROMPT}${SECTION}What type of MMU are you running?${INPUT}"
-    echo -e "1) ERCF v1.1 (inc TripleDecky, Springy, Binky mods)"
-    echo -e "2) ERCF v2.0"
-    echo -e "3) Tradrack v1.0"
-    echo -e "4) Angry Beaver v1.0"
-    echo -e "5) Armored Turtle v1.0"
-    echo -e "6) 3MS (Modular Multi Material System) v1.0"
-    echo -e "7) Other / Custom (or just want starter config files)"
-    num=$(prompt_123 "MMU Type?" 7)
-    echo
-    case $num in
-        1) # ERCF v1.1
+    options=(
+        'ERCF v1.1 (inc TripleDecky, Springy, Binky mods)'
+        'ERCF v2.0'
+        'Tradrack v1.0'
+        'Angry Beaver v1.0'
+        'Armored Turtle v1.0'
+        '3MS (Modular Multi Material System) v1.0'
+        'Other / Custom (or just want starter config files)'
+    )
+    prompt_option opt 'MMU Type' "${options[@]}"
+    case $opt in
+        *'ERCF v1.1'*)
```

Comment:
I'm on the fence between matching the text vs the number. Using this `prompt_option` we can do either one. If we want to match by number we would change the case to use `$REPLY` instead of `$opt`.

```shell
case $REPLY in
    1)
        # ...
```

Matching by text makes it less reliant order which could be useful for dynamic option lists but potentially prone to typo errors. Matching by item number is more reliant on the order of the options. Any preference one way or another? Happy to update these if we want to match by number instead...just let me know what you'd prefer.